templates/web/base/report/update.html

```diff
@@ -1,7 +1,7 @@
 [% can_moderate = NOT update.whenanswered AND update.type != 'moderation' AND (permissions.moderate OR c.user.can_moderate(update, staff = permissions.moderate)) %]
 [% IF loop.first %]
 <section class="full-width">
-    <h4 class="static-with-rule">[% loc('Updates') %]</h4>
+    <h2 class="static-with-rule">[% loc('Updates') %]</h2>
```

Comment:
The fixamingata and oxfordshire layout.scss also have a h4 that will need changing